example/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
```

Comment:
@stephenkingsley no point of having a seperate example, when we already have demo

Reply:
we can temporarily remove the CDN link in demo for testing purposes